src/routes/redirects.ts

```diff
@@ -21,32 +16,22 @@ const sendAnalyticsEvent = async (
     const query = req.query as Record<string, string>;
     const queryStr = JSON.stringify(query);
     const now = new Date();
-    const events = [
+    await sendAnalyticsEvent([
       {
         event_timestamp: now,
-        event_id: generateEventId(now),
         event_name: 'page view',
         event_page: '/get',
         app_platform: 'redirector',
         query_params: queryStr.length > 2 ? queryStr : undefined,
-        session_id: boot.visit.sessionId,
```

Comment:
There shouldnt be any linking of boot ids to tracking?
These related elements would all get new ids correct?

Reply:
Nope, I just unified the logic here and moved it internally